src/client/reducers/reducer_filters.js

```diff
@@ -0,0 +1,29 @@
+const dafaultFilters = [{
```

Comment:
@vitkuz if i understand it correctly you store a reducer a model of a filter component? So you use it render a filter component, right? why don't we move this logic into the component itself

Filters are used to do a sorting, right?
So component having movies data can generate a proper listing of movies. 
Keeping it in movies is a bit excessive.

Reply:
@vitkuz in case you use filters for search you can simply do a search with an active filter type.

Reply:
@algbeta Filters are not related to search. They a related to sorting items